config/environments/production.rb

```diff
@@ -32,7 +32,7 @@
   config.logstasher.suppress_app_log = true
 
   # Defaults to Rails.root.join("public/assets")
-  # config.assets.manifest = YOUR_PATH
+  config.assets.manifest = Rails.root.join("public/government/assets/assets-manifest.json")
```

Comment:
Thanks, that makes sense.  It might be worth updating this comment when we come back to tidy the whole thing up.